maven-plugins/enforcer-maven-plugin/src/main/java/io/helidon/build/maven/enforcer/GitIgnore.java

```diff
@@ -144,7 +144,8 @@ private boolean isParentExcluded(String pattern) {
         pattern = "/" + pattern;
         Path parent = Path.of(pattern).getParent();
         if (!parent.endsWith("/")) {
-            pattern = parent + "/";
+            //if Windows path then replace the separator
```

Comment:
A fix has been merged for this already.
Please rebase your branch to pickup the changes.